From 371a986adc5c4a576eacb56d7fc2e7d510585404 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Wed, 24 Aug 2016 10:40:53 -0700
Subject: [PATCH] Disable excessive audit logs

These logs are spamming kmsg way too hard, destroying kernel logs and
making kernel debugging difficult, so disable them.

Change-Id: Idb2281f354187b8076daec0c9e0013eefdb1c54a
---
 init/log.cpp      | 2 ++
 logd/LogAudit.cpp | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/init/log.cpp b/init/log.cpp
index d32f2da..d965d7b 100644
--- a/init/log.cpp
+++ b/init/log.cpp
@@ -48,6 +48,7 @@ void init_klog_write(int level, const char* fmt, ...) {
 }
 
 int selinux_klog_callback(int type, const char *fmt, ...) {
+#if 0
     int level = KLOG_ERROR_LEVEL;
     if (type == SELINUX_WARNING) {
         level = KLOG_WARNING_LEVEL;
@@ -58,5 +59,6 @@ int selinux_klog_callback(int type, const char *fmt, ...) {
     va_start(ap, fmt);
     init_klog_vwrite(level, fmt, ap);
     va_end(ap);
+#endif
     return 0;
 }
diff --git a/logd/LogAudit.cpp b/logd/LogAudit.cpp
index 808d17a..a63fefa 100644
--- a/logd/LogAudit.cpp
+++ b/logd/LogAudit.cpp
@@ -67,8 +67,10 @@ bool LogAudit::onDataAvailable(SocketClient *cli) {
         return false;
     }
 
+#if 0
     logPrint("type=%d %.*s",
         rep.nlh.nlmsg_type, rep.nlh.nlmsg_len, rep.data);
+#endif
 
     return true;
 }
-- 
2.7.4

